src/saga/adaptors/torque/torquejob.py

```diff
@@ -867,17 +821,15 @@ def _job_get_info(self, job_id, reconnect):
                     val = val.strip()
 
                     # The ubiquitous job state
-                    if key in ['job_state']: # PBS Pro and TORQUE
-                        job_info['state'] = _torque_to_saga_jobstate(val)
+                    if key in ['job_state']:
+                        job_state = val
 
```

Comment:
You are right, good catch!
The mapping needs to go to line 825 indeed.

Alternatively I could move this all inside _torque_to_saga_jobstate by adding return_val to its parameters?